From: Debian Python Modules Team
 <python-modules-team@lists.alioth.debian.org>
Date: Tue, 15 Jan 2019 00:39:20 -0500
Subject: no_vendored_html5lib

Updated for bleach 3.2.0 by Leo <thinkabit.ukim@gmail.com>
Updated for bleach 3.2.1 by Leo <thinkabit.ukim@gmail.com>

===============================================================================
diff --git a/bleach/html5lib_shim.py b/bleach/html5lib_shim.py
index 7bfd581..c4c389d 100644
--- a/bleach/html5lib_shim.py
+++ b/bleach/html5lib_shim.py
@@ -18,43 +18,43 @@ warnings.filterwarnings(
     "ignore",
     message="html5lib's sanitizer is deprecated",
     category=DeprecationWarning,
-    module="bleach._vendor.html5lib",
+    module="html5lib",
 )
 
-from bleach._vendor.html5lib import (  # noqa: E402 module level import not at top of file
+from html5lib import (  # noqa: E402 module level import not at top of file
     HTMLParser,
     getTreeWalker,
 )
-from bleach._vendor.html5lib import (
+from html5lib import (
     constants,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib.constants import (  # noqa: E402 module level import not at top of file
+from html5lib.constants import (  # noqa: E402 module level import not at top of file
     namespaces,
     prefixes,
 )
-from bleach._vendor.html5lib.constants import (
+from html5lib.constants import (
     _ReparseException as ReparseException,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib.filters.base import (
+from html5lib.filters.base import (
     Filter,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib.filters.sanitizer import (
+from html5lib.filters.sanitizer import (
     allowed_protocols,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib.filters.sanitizer import (
+from html5lib.filters.sanitizer import (
     Filter as SanitizerFilter,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib._inputstream import (
+from html5lib._inputstream import (
     HTMLInputStream,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib.serializer import (
+from html5lib.serializer import (
     HTMLSerializer,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib._tokenizer import (
+from html5lib._tokenizer import (
     attributeMap,
     HTMLTokenizer,
 )  # noqa: E402 module level import not at top of file
-from bleach._vendor.html5lib._trie import (
+from html5lib._trie import (
     Trie,
 )  # noqa: E402 module level import not at top of file
 
diff --git a/tests/test_clean.py b/tests/test_clean.py
index 1cd58df..a652fcf 100644
--- a/tests/test_clean.py
+++ b/tests/test_clean.py
@@ -7,7 +7,7 @@ import pytest
 from bleach import clean
 from bleach.html5lib_shim import Filter
 from bleach.sanitizer import Cleaner
-from bleach._vendor.html5lib.constants import rcdataElements
+from html5lib.constants import rcdataElements
 
 
 def test_clean_idempotent():
